x-pack/plugins/security_solution/common/detection_engine/build_exceptions_query.test.ts

```diff
@@ -137,14 +64,14 @@ describe('build_exceptions_query', () => {
     describe('kuery', () => {
       test('it returns formatted wildcard string when operator is "excluded"', () => {
         const query = buildExists({
-          entry: existsEntryWithExcluded,
+          entry: { ...getEntryExistsMock(), operator: 'excluded' },
```

Comment:
I do like the named constants in the general case to help answer "what does this object represent?" but in this case that's mostly redundant with the test description 👍